tests/com/twitter/intellij/pants/execution/OSSPantsJvmRunConfigurationIntegrationTest.java

```diff
@@ -20,48 +20,41 @@
 import com.intellij.psi.PsiElement;
 import com.intellij.psi.PsiMethod;
 import com.intellij.psi.PsiPackage;
-import com.intellij.psi.search.GlobalSearchScope;
 import com.intellij.testFramework.MapDataContext;
 import com.twitter.intellij.pants.PantsManager;
+import com.twitter.intellij.pants.util.ProjectTestJvms;
 import com.twitter.intellij.pants.service.task.PantsTaskManager;
 import com.twitter.intellij.pants.settings.PantsExecutionSettings;
 import com.twitter.intellij.pants.testFramework.OSSPantsIntegrationTest;
 import com.twitter.intellij.pants.util.PantsUtil;
 import org.jetbrains.annotations.NotNull;
+import org.junit.Assert;
 
 import java.util.Arrays;
+import java.util.Collection;
 import java.util.Collections;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Optional;
 import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 
 public class OSSPantsJvmRunConfigurationIntegrationTest extends OSSPantsIntegrationTest {
   public void testClassRunConfiguration() throws Throwable {
     doImport("testprojects/tests/java/org/pantsbuild/testproject/testjvms");
 
-    String classReference = "org.pantsbuild.testproject.testjvms.TestSix";
-
-    PsiClass testClass = JavaPsiFacade.getInstance(myProject).findClass(classReference, GlobalSearchScope.allScope(myProject));
-    assertNotNull(testClass);
+    PsiClass testClass = ProjectTestJvms.anyTestClass(myProject, getProjectPath());
 
     ExternalSystemRunConfiguration esc = getExternalSystemRunConfiguration(testClass);
 
     // Make sure task name is `test` goal.
-    assertEquals(Collections.singletonList("test"), esc.getSettings().getTaskNames());
+    String testTask = "test";
+    assertEquals(Collections.singletonList(testTask), esc.getSettings().getTaskNames());
 
     List<String> configScriptParameters = PantsUtil.parseCmdParameters(Optional.ofNullable(esc.getSettings().getScriptParameters()));
-
-    List<String> expectedConfigScriptParameters = Arrays.asList(
-      "testprojects/tests/java/org/pantsbuild/testproject/testjvms:eight-test-platform",
-      "testprojects/tests/java/org/pantsbuild/testproject/testjvms:six",
-      "testprojects/tests/java/org/pantsbuild/testproject/testjvms:seven",
-      "testprojects/tests/java/org/pantsbuild/testproject/testjvms:eight",
-      "testprojects/tests/java/org/pantsbuild/testproject/testjvms:base",
-      "--test-junit-test=" + classReference
-    );
-    assertEquals(expectedConfigScriptParameters, configScriptParameters);
```

Comment:
Do you think it'd be possible to wrap this around


```
if (PantsUtil.isCompatiblePantsVersion(getProjectPath(), "1.25.0")) { 
...
} 
else {
// testprojects/tests/java/org/pantsbuild/testproject/testjvms:base
// testprojects/tests/java/org/pantsbuild/testproject/testjvms:eight
// testprojects/tests/java/org/pantsbuild/testproject/testjvms:eight-test-platform
}

```

Reply:
Sorry, I don't think I follow. 
The highlighted code is removed from repository as it is no longer necessary - now, we don't use hardcoded names of the files but rather inspect the directory and get expected names from there. This should work the same way regardless of the pants version.

Reply:
I see. I guess that works, too. Thanks!

My original thought was to keep the existing mechanism, and assert on different targets based on the pants version in the pantsbuild/pants repo.